Modules/System/Location/app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "7f03f010-1703-47eb-a3cd-c6d28b8a806c",
+  "name": "Location",
```

Comment:
Location is a "reserved" word in BC. Reusing location for an entirely different concept is risky, as it might create confusion. We should think of a different name. Suggestion: GeoLocation - or something along these lines.

Reply:
I renamed the module and functions to "GeoLocation" instead of "Location".